packages/app-project/src/helpers/getDefaultPageProps/getDefaultPageProps.js

```diff
@@ -8,24 +7,15 @@ const HOSTS = {
   staging: 'https://frontend.preview.zooniverse.org'
 }
 
-export default async function getDefaultPageProps({ params, query, req }) {
-
-  // cookie is in the next.js context req object
-  const mode = getCookie(req, 'mode') || null
-  const dismissedAnnouncementBanner = getCookie(req, 'dismissedAnnouncementBanner') || null
```

Comment:
This has probably never worked, because the store property is called `dismissedProjectAnnouncementBanner`.